keras_nlp/layers/positional_embedding.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2022 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Positional embedding implementation based on `keras.layers.Layer`."""
+
+import tensorflow as tf
+from tensorflow import keras
+
+
+class PositionalEmbedding(keras.layers.Layer):
+    """Creates a positional embedding.
+
+    Args:
+      max_length: The maximum size of the dynamic sequence.
+      initializer: The initializer to use for the embedding weights. Defaults to
+        "glorot_uniform".
+      seq_axis: The axis of the input tensor where we add the embeddings.
+
+    Example:
+    ```python
+    positional_embedding = PositionalEmbedding(max_length=100)
```

Comment:
Show the full symbol `keras_nlp.layers.PositionalEmbedding`.

I would also show this example alone with a `keras.layers.Embedding` and a sum. That would help explain the use case better.

Reply:
I was waiting for us to have a the positional embedding layer ready to do this (I was assuming we would port the OnDeviceEmbedding one over). But yeah, since we'll be using the Keras one, I'll add the example.

Reply:
Will add such an example